### DISCUSSION TITLE
Multi-select details question

### DISCUSSION THREAD

**Discussion by okdistribute** [digidem/mapeo-mobile#826] [Ideas]
– currently the user can add details to an observation. We support two question types: text area and single-select
- Add a multi-select question type, showing options as checkboxes rather than radio buttons
- Preset schema is defined in https://github.com/digidem/mapeo-schema"

**Comment by maccomaccomaccomacco** [comment]
I would developer this feature: should the multi-select replace the single-select?

**Comment by okdistribute** [comment]
@marcoautiero wow that would be great marco, thanks for offering to help! No, the multi-select option is aiming to be an additional feature.
